crates/matrix-sdk-crypto/src/store/mod.rs

```diff
@@ -54,12 +54,10 @@ use std::{
     sync::Arc,
 };
 
-use base64::DecodeError;
+#[cfg(feature = "indexeddb_cryptostore")]
+use indexed_db_futures::web_sys::DomException;
```

Comment:
what is this needed for?

Reply:
It isn't. This PR was made before the big store changes happened, so this seems to be a leftover from a merge/rebase. Removed in https://github.com/matrix-org/matrix-rust-sdk/pull/494/commits/e7d0ac42ef8634fa8f79667b0dc3ef34df51ed30.